package.json

```diff
@@ -54,5 +54,28 @@
     "trailingComma": "es5",
     "singleQuote": true,
     "printWidth": 120
+  },
+  "devDependencies": {
```

Comment:
Are all of these necessary? The tests seem to work fine after I remove the following dependencies:

```
"@typescript-eslint/eslint-plugin": "^5.19.0",
"@typescript-eslint/parser": "^5.19.0",
"eslint": "^8.13.0",
"eslint-config-prettier": "^8.5.0",
"eslint-config-standard": "^16.0.3",
"eslint-plugin-import": "^2.26.0",
"eslint-plugin-node": "^11.1.0",
"eslint-plugin-prettier": "^4.0.0",
"eslint-plugin-promise": "^6.0.0",
"prettier": "^2.6.2",
"prettier-plugin-solidity": "^1.0.0-beta.19",
"solhint": "^3.3.7",
```

After removing these, I get only the following unmet peer dependencies:

```
warning " > @typechain/hardhat@6.0.0" has unmet peer dependency "@ethersproject/abi@^5.4.7".
warning " > @typechain/hardhat@6.0.0" has unmet peer dependency "@ethersproject/providers@^5.4.7".
warning " > @typechain/ethers-v5@10.0.0" has unmet peer dependency "@ethersproject/abi@^5.0.0".
warning " > @typechain/ethers-v5@10.0.0" has unmet peer dependency "@ethersproject/bytes@^5.0.0".
warning " > @typechain/ethers-v5@10.0.0" has unmet peer dependency "@ethersproject/providers@^5.0.0"
```